config/v3_0_experimental/config.go

```diff
@@ -32,28 +32,19 @@ func Parse(rawConfig []byte) (types.Config, report.Report, error) {
 		return types.Config{}, report.Report{}, errors.ErrEmpty
 	} else if isCloudConfig(rawConfig) {
 		return types.Config{}, report.Report{}, errors.ErrCloudConfig
-	} else if isScript(rawConfig) {
```

Comment:
The script stuff is part of cloud-init support, I'm afraid.  :open_mouth:

Reply:
Is that true for upstream cloud-init? I can't seem to find docs for it. If it's only coreos-cloudinit that supports it we should still drop it.

Reply:
[Yes](https://cloudinit.readthedocs.io/en/latest/topics/format.html#user-data-script).  I just meant that this change should be moved to the cloudinit removal commit.

Reply:
Fixed now.